doc/source/serve/doc_code/visualize_dag_during_deployment.py

```diff
@@ -0,0 +1,37 @@
+import ray
+from ray import serve
+from ray.serve.deployment_graph import InputNode
+
+ray.init()
+
+
+@serve.deployment
+class Model:
+    def __init__(self, weight):
+        self.weight = weight
+
+    def forward(self, input):
+        return input + self.weight
+
+
+@serve.deployment
+def combine(output_1, output_2, kwargs_output=0):
+    return output_1 + output_2 + kwargs_output
+
+
+with InputNode() as user_input:
+    m1 = Model.bind(1)
+    m2 = Model.bind(2)
+    m1_output = m1.forward.bind(user_input[0])
+    m2_output = m2.forward.bind(user_input[1])
+    dag = combine.bind(m1_output, m2_output, kwargs_output=user_input[2])
+
+# Partial DAG visualization
+graph = ray.experimental.dag.vis_utils.dag_to_dot(m1_output)
```

Comment:
non blocking, `vis_utils` should be renamed for more user friendly naming like `visualization`.

Reply:
the API for this is aligned with our earlier agreement of `ray.dag.run()` with `ray.dag.plot`: https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/ray-project/ray%24+ray.experimental.dag.plot&patternType=literal

but i think surfacing the `to_dot` at this level can be helpful too for non-jupyter users. Noted :)